### ISSUE TITLE
typescript compling error

### ISSUE THREAD

**Issue by joesonw** [mixer/etcd3#54]
node_modules/etcd3/lib/src/connection-pool.d.ts(16,98): error TS2694: Namespace '"grpc"' has no exported member 'ChannelOptions'.
node_modules/etcd3/lib/src/options.d.ts(2,10): error TS2305: Module '"grpc"' has no exported member 'ChannelOptions'.
node_modules/etcd3/lib/src/rpc.d.ts(13,43): error TS2694: Namespace '"grpc"' has no exported member 'StatusMessage'.

etcd3: 0.2.7
typescript: 2.6.2

**Comment by connor4312** [comment]
Thanks, I'll take a look at this this weekend.

**Comment by connor4312** [comment]
Unable to repro this on TS 2.7, and Travis builds fine. Marking as resolved.

**Comment by Eatingtec** [comment]
I still got the same error on TS 2.7.2( by tsc my code which import Etcd3 from npm install).

But I copy the file (etcd3/src/types/grpc.d.ts) from git to my project (path is src/types) and 
change the paths config in tsconfig.json as follows:
"paths": {
      "*": [
        "src/types/*"
      ]
    }

finally, it works, Error no longer occurs

**Comment by willgm** [comment]
@connor4312,  I'm getting the same error at this repo:

https://github.com/willgm/express-session-etcd3

After install the dependencies just run `npm run build` to see the follow error message:

```
~/proj/express-session-etcd3 [master *] » npm run build                                                                                                                                           willgm@localhost

> express-session-etcd3@0.0.0 prebuild /home/willgm/proj/express-session-etcd3
> rimraf dist


> express-session-etcd3@0.0.0 build /home/willgm/proj/express-session-etcd3
> tsc --module commonjs && rollup -c rollup.config.ts && typedoc --out docs --target es6 --theme minimal --mode file src

node_modules/etcd3/lib/src/connection-pool.d.ts(16,98): error TS2694: Namespace '"grpc"' has no exported member 'ChannelOptions'.
node_modules/etcd3/lib/src/options.d.ts(2,10): error TS2305: Module '"grpc"' has no exported member 'ChannelOptions'.
node_modules/etcd3/lib/src/rpc.d.ts(13,43): error TS2694: Namespace '"grpc"' has no exported member 'StatusMessage'.
node_modules/grpc/index.d.ts(2,36): error TS7016: Could not find a declaration file for module 'protobufjs'. '/home/willgm/proj/express-session-etcd3/node_modules/protobufjs/dist/protobuf.js' implicitly has an 'any' type.
  Try `npm install @types/protobufjs` if it exists or add a new declaration (.d.ts) file containing `declare module 'protobufjs';`
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! express-session-etcd3@0.0.0 build: `tsc --module commonjs && rollup -c rollup.config.ts && typedoc --out docs --target es6 --theme minimal --mode file src`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the express-session-etcd3@0.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/willgm/.npm/_logs/2018-03-02T21_25_28_846Z-debug.log
```

**Comment by connor4312** [comment]
Thanks for the followup. Setting `skipLibCheck` should fix this, I'll fix it in this repo though.

**Comment by luisw19** [comment]
@connor4312  I am also getting a similar error. However when trying the workarounds listed here (skipLibCheck and paths) I then get a different error

```
internal/modules/cjs/loader.js:613
    throw err;
    ^
Error: Cannot find module 'xxxxx/dist/server.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:610:15)
    at Function.Module._load (internal/modules/cjs/loader.js:526:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:824:10)
    at internal/main/run_main_module.js:17:11 
```

**Comment by edvardchen** [comment]
Any progress?

**Comment by dusktreader** [comment]
Wondering why this was closed. Still getting this issue

**Comment by jwalton** [comment]
It was fixed by #108, but a new version hasn't been published to npm yet.
